app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -110,6 +112,17 @@ public void onClickContributor(String htmlUrl) {
         intentOptional.ifPresent(this::startActivity);
     }
 
+    @Override
+    public void onSwipeRefresh() {
+        Disposable disposable = viewModel.getContributors(true)
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                        this::renderContributors,
+                        throwable -> Timber.tag(TAG).e(throwable, "Failed to show contributors.")
+                );
+        compositeDisposable.add(disposable);
```

Comment:
I have found another idea to use `ObservableList` in databinding package.
https://developer.android.com/reference/android/databinding/ObservableList.html

I will try either DataUtil or ObservableList on another PR so please ignore this time.